samples/peripheral/radio_test/src/radio_cmd.c

```diff
@@ -1114,6 +1229,31 @@ SHELL_STATIC_SUBCMD_SET_CREATE(sub_transmit_pattern,
 	SHELL_SUBCMD_SET_END
 );
 
+#if CONFIG_NRF21540_FEM
+SHELL_STATIC_SUBCMD_SET_CREATE(sub_nrf21540_antenna,
+	SHELL_CMD(ant_1, NULL,
```

Comment:
If you use SHELL_CMD_ARG you can specify the argument count, and don't have to do it in your command handlers.

Reply:
I did some try with `SHELL_CMD_ARG`. I would not use it here since the commands are then not displayed as I would like when using it